### PR TITLE
Complete App

### PR DESCRIPTION
**Pull request by DuaneDave**
### In this PR, I implemented the cashback app with the following features:

[Live presentation](https://www.loom.com/share/c63597a66c6f4f82bc48018c234be8f7)

- Login, sign up and splash screen
- Category Page
- Transaction page
- Wrote test for the app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,90 +148,68 @@ Example:
...
+If you like this project buy the team a [drink](.../.../)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (Davidosky007) [inline README.md:189]: * Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -1,15 +1,290 @@
...
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: normal;
+  font-family: 'Roboto', sans-serif;
```
Comment (Davidosky007) [inline app/assets/stylesheets/application.css:34]: * Kindly use the right design as stated in the [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) 

> * Colors

> * Typography: font face, size, and weight.
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @DuaneDave 👋, 

Good job so far!
There are some issues that
 you still need to work on to go to the next project but you are almost there!
Almost-gif-simpson
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Professional video presentation 👍 
* All tests passed ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/assets/stylesheets/application.css (hunk 2 above)
Comment (DuaneDave) [reply app/assets/stylesheets/application.css:34]: Thank you @Davidosky007 for the feedback. I want to let you know that this font is not a free font and it is not available in google fonts hence I am using a substitute font and as per the colour, I am using the same colour scheme and they are all in a CSS custom variable.

**Review by DeliceLydia — APPROVED** [review]
## Hi @DuaneDave, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**